sysutils/spacebar/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               makefile 1.0
+PortGroup               xcodeversion 1.0
+
+github.setup            cmacrae spacebar 1.4.0 v
+
```

Comment:
if upstream provides "releases" you would ideally set `github.tarball_from releases`; if not the next preferred option is `github.tarball_from archive`. You will need to update the `checksums` when making that change.